hooks/node_handlers/base_export_handler.py

```diff
@@ -0,0 +1,644 @@
+        :param publish_template: The publish :class:`sgtk.Template` for this parm.
+        :param list paths_and_templates: The current list of paths and templates
+            to append to.
+        :param bool is_deep: Is the parm a deep output or not.
+        """
+        parm = node.parm(parm_name)
+        path = parm.evalAsString()
+        
+        item = {
+            "work_template": work_template,
+            "publish_template": publish_template
+        }
+        fields = work_template.get_fields(path)
+        
+        sequence_paths = self._get_sequence_paths(path, work_template, fields)
+        if sequence_paths:
+            item["sequence_paths"] = sequence_paths
+        elif not os.path.exists(path):
+            # if nothing exists on disk
+            # lets get the last rendered one because it might be "<Next>"
+            # and it's evaluated to the next available version
+            all_versions = self._get_all_versions(node)
+            if not all_versions:
+                # no versions so nothing's been rendered
+                return
+            fields["version"] = max(all_versions)
+            path = work_template.apply_fields(fields)
+            sequence_paths = self._get_sequence_paths(path, work_template, fields)
+            if sequence_paths:
+                item["sequence_paths"] = sequence_paths
+
+        item["path"] = self._make_sgtk_compliant_path(path, work_template)
+
+        if is_deep:
+            item["is_deep"] = True
+        paths_and_templates.append(item)
+
+    def _get_output_paths_and_templates(self, node):
+        """
+        Go through the node's specified parameters and get the output paths,
+        work and publish templates.
+
+        Returns a list of dictionaries, each containing, at least:
+        - work template
+        - publish template
+        - file name
+
+        and optionally:
+        - any sequence paths
+        - whether the output is a deep image
+
+        :param node: A :class:`hou.Node` instance.
+
+        :rtype: list(dict)
+        """
+        paths_and_templates = []
+
+        work_template = self.get_work_template(node)
+        publish_template = self.get_publish_template(node)
+        self._get_output_path_and_templates_for_parm(
+            node,
+            self.OUTPUT_PARM,
+            work_template,
+            publish_template,
+            paths_and_templates
+        )
+        
+        return paths_and_templates
+
+    def get_output_paths_and_templates(self, node):
+        """
+        Go through the node's specified parameters and get the output paths,
+        work and publish templates.
+
+
+        :param node: A :class:`hou.Node` instance.
+
+        :rtype: list(dict)
+        """
+        parm = node.parm(self.USE_SGTK)
+        if not (parm or parm.eval()):
+            return []
+        
+        return self._get_output_paths_and_templates(node)
```

Comment:
The `parm.eval()` will run if `parm` evaluates to `False`, which  would probably break if `parm is None`

Maybe try

```suggestion
        results = []
        if parm and parm.eval():
            results = self._get_output_paths_and_templates(node)
        return results
```